flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/testutils/RestoreTestBase.java

```diff
@@ -117,6 +122,11 @@ public EnumSet<TestKind> supportedRunSteps() {
         return EnumSet.of(TestKind.SQL);
     }
 
+    @BeforeEach
+    public void clearData() {
```

Comment:
Nice find!

I think we should be unified with how it's done at other places. `JsonPlanTestBase` does it in `AfterEach`. If we reuse a name between those two, `JsonPlanTestBase` can fail if executed after `RestoreTestBase`.

Could we also clear the `localRawResultsObservers` here?

Reply:
A food for thought. Would it make sense to add some unique identifier to all the sinks/sources in `RestoreTestBase`? We could use e.g. the test id or generate some UUID for the duration of a test run.

Reply:
>A food for thought. Would it make sense to add some unique identifier to all the sinks/sources in RestoreTestBase? We could use e.g. the test id or generate some UUID for the duration of a test run.

I thought about trying to do that.  Seems like we'd have to do it manually in the test cases or update the SQL somehow if we were doing it in the code.  I didn't see an easy way to sort that out.